client/my-sites/domains/domain-management/contacts-privacy/index.jsx

```diff
@@ -55,31 +58,40 @@ const ContactsPrivacy = React.createClass( {
 						hasPrivacyProtection={ hasPrivacyProtection }
 						privateDomain={ privateDomain }
 						privacyAvailable={ privacyAvailable }
-						currentUserCanManage={ currentUserCanManage } />
+						currentUserCanManage={ currentUserCanManage }
+					/>
 
 					<VerticalNavItem
-							path={ paths.domainManagementEditContactInfo( this.props.selectedSite.slug, this.props.selectedDomainName ) }>
-						{ this.translate( 'Edit Contact Info' ) }
+						path={ paths.domainManagementEditContactInfo(
+							this.props.selectedSite.slug,
+							this.props.selectedDomainName
+						) }
+					>
+						{ translate( 'Edit Contact Info' ) }
 					</VerticalNavItem>
 
 					{ ! hasPrivacyProtection && privacyAvailable && (
 						<VerticalNavItem
-							path={ paths.domainManagementPrivacyProtection( this.props.selectedSite.slug, this.props.selectedDomainName ) }>
-							{ this.translate( 'Privacy Protection' ) }
+							path={ paths.domainManagementPrivacyProtection(
+								this.props.selectedSite.slug,
+								this.props.selectedDomainName
+							) }
+						>
+							{ translate( 'Privacy Protection' ) }
 						</VerticalNavItem>
 					) }
 				</VerticalNav>
 			</Main>
 		);
-	},
+	}
 
 	isDataLoading() {
```

Comment:
Sure, you bind the functions that you pass in child components so when called they are actually bound to the parent. :)

When you call `this.something()` no need to bind, as you're calling it as a class method.